src/_posts/2022-05-17-restructuring-openoakland.md

```diff
@@ -0,0 +1,51 @@
+
+Although this vote may *seem* like a formality, this process helps us maintain accountability, ensure we're adhering to the spirit of our bylaws, and maintain paths to participation for volunteers interested in reshaping OpenOakland.
+
+## How much effort does it really take to run OpenOakland?
+
+There is a surprising amount of labor that goes into running the brigade. Onboarding new volunteers and matching them to appropriate opportunities requires one-on-one attention. Scheduling, structuring, and facilitating meetups every week requires hours of prep before the event even starts. Helping to manage and access accounts, documentation, and subscription services for multiple projects requires organization and communication with different people on different schedules. Communicating out to the public about brigade and project team activities requires hours of writing and publishing. And then there's the continuous effort of supporting member requests, building relationships with community partners, navigating interpersonal relationships, etc. Most of this is *good work* that helps us all have more of an impact. But it can be challenging for volunteers to do it all on top of our day jobs and personal lives.
+
+Because Project Leads tend to be the ones representing their team on Steering Committee, Leads end up stretched thin and unable to manage everything. So tasks slip through the cracks and over time and the process and infrastructure that holds the brigade together begins to fray. Larger initiatives like project lifecycle planning, impact measurement, recruiting strategies, and partnerships fall by the wayside. As we've seen, the result is exhaustion and volunteer attrition over time.
+
+## What is "Lean Ops"?
+
+Our Lean Ops (lean operations) mode is intended to strip operational tasks down to the very barebones needed to accomplish two core goals:
+
+1. Maintain official brigade status with Code for America.
+2. Improve the sustainable participation, representation, and decision-making among Oaklanders, particularly those historically underrepresented in tech, through the restructuring of our bylaws, decision-making processes, and community rituals.
+
+To do this, the existing Steering Committee (made up of active project leads) will share responsibility for the following limited activities:
+
+- **Host our regular Steering Committee meeting** on the third Tuesday of each month.
+- **Schedule special Steering Committee meetings as needed** to continue the restructuring work.
+- **Participate in Code for America's ongoing Direct Democracy pilot** through a smaller cohort of 3-4 members tasked with bringing learnings and recommendations back to Steering Committee.
+- **Host member meetings, workshops, and votes as needed** to inform the restructuring process.
+- **Post public updates on our progress** to the openoakland.org blog and through our email newsletter.
+- **Manage expenses** for existing project commitments.
+
+### What we won't do
```

Comment:
How about "What we're putting on hold/pause/ice"?

Reply:
Love this change.